src/views/sessions/CommandList/BaseList.vue

```diff
@@ -39,6 +40,7 @@ export default {
     const dateFrom = getDaysAgo(7).toISOString()
     const dateTo = this.$moment(getDayEnd()).add(1, 'day').toISOString()
     return {
+      title: this.$t('CommandStorage'),
       loading: true,
       tableConfig: {
         url: '/api/v1/terminal/commands/',
```

Comment:
It seems like you've posted a single line of code with no context or details to discuss about potential differences, irregularities, issues, optimizations etc. Please provide more information so that I can address your query accordingly.

Here’s what it looks like:

```javascript
TreeTable(
  ref="CommandTreeTable",
  data={{ ...}}
)
```
What is unclear from this simple snippet?